virttest/standalone_test.py

```diff
@@ -56,12 +57,6 @@ def find_default_qemu_paths(options_qemu=None, options_dst_qemu=None):
     qemu_img_path = os.path.join(qemu_dirname, 'qemu-img')
     qemu_io_path = os.path.join(qemu_dirname, 'qemu-io')
 
-    if not os.path.exists(qemu_img_path):
```

Comment:
it looks qemu-img,qemu-io and qemu-kvm  not always in same directory, eg:

[root@dhcp-10-29]# ls -l /usr/libexec/qemu-kvm 
-rwxr-xr-x 1 root root 7882208 Jun  6 18:41 /usr/libexec/qemu-kvm
~
[root@dhcp-10-29]# which qemu-img 
/usr/bin/qemu-img
~
[root@dhcp-10-29]# which qemu-io
/usr/bin/qemu-io

qemu-img-rhev-2.6.0-5.el7.x86_64
qemu-kvm-rhev-2.6.0-5.el7.x86_64
qemu-kvm-tools-rhev-2.6.0-5.el7.x86_64
So, I think it's better to get qemu-io and qemu-img full path via it use utils_path.find_command.